stock_package_info/models/product_packaging_template.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.addons import decimal_precision as dp
+
+
+class ProductPackagingTemplate(models.Model):
+    _name = 'product.packaging.template'
+    _description = 'Product Packaging Template'
+    _rec_name = 'packaging_template_name'
+
+    packaging_template_name = fields.Char(required=True)
+    package_type = fields.Selection([
+        ('unit', 'Unit'),
+        ('pack', 'Pack'),
+        ('box', 'Box'),
+        ('pallet', 'Pallet'),
+    ],
+        required=True,
+    )
+    product_tmpl_id = fields.Many2one(
+        comodel_name='product.template',
+        string='Product',
+        index=True,
+        ondelete='cascade',
+    )
+    length = fields.Float(
```

Comment:
Hey all,

In https://github.com/OCA/stock-logistics-workflow/pull/247/commits/8151c7b882fe48911e6995e993e383baf7d4ab29 I updated all the measurement field names by prepending `pack_` to them as I thought only updating the `length`  field name would create a weird incongruity in naming. Does anyone oppose this change?

Thanks!